azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/CallComposite.java

```diff
@@ -137,6 +151,33 @@ public void launch(final Context context,
         launchComposite(context, remoteOptions, localOptions, false);
     }
 
+    /**
+     * Handle incoming call push notification.
+     *
+     * <pre>
+     *
+     * final CommunicationTokenRefreshOptions communicationTokenRefreshOptions =
+     *                 new CommunicationTokenRefreshOptions&#40;tokenRefresher, true&#41;;
+     * final CommunicationTokenCredential credential =
+     *                 new CommunicationTokenCredential&#40;communicationTokenRefreshOptions&#41;;
+     * final CallCompositePushNotificationInfo pushNotificationInfo =
+     *                 new CallCompositePushNotificationInfo&#40;notificationMap&#41;;
+     * final CallCompositeRemoteOptions remoteOptions =
+     *                 new CallCompositeRemoteOptions&#40;pushNotificationInfo, credential, displayName&#41;;
+     * callComposite.launch&#40;context, remoteOptions&#41;;
+     *
+     * </pre>
+     *
+     * @param context           The android context used to start the Composite.
+     * @param remoteOptions     The {@link CallCompositeRemoteOptions} has remote parameters to
+     *                              launch group call experience.
+     */
+    public void handlePushNotification(final Context context,
```

Comment:
naming here is a bit confusing, it's called handlePushNotification, but it doesn't actually take anything directly related push as a parameter.

Also the method it delegates to (overrides params) is pretty far in the file, it'd be nice if they were side by side.

Reply:
As per discussion we will fix for GA.